engine/classes/Elgg/Database/EntityTable.php

```diff
@@ -1221,7 +1196,13 @@ public function getEntityAttributeWhereSql(array $options = []) {
 		}
 
 		// @todo the types should be defined somewhere (as constant on \ElggEntity?)
-		if (!in_array($type, ['group', 'object', 'site', 'user'])) {
+		if (!in_array($type, [
```

Comment:
If our CS rules think this is better I have words for it. Maybe make a $valid_types array above and use it?